tests/Unit/ControlSystem/ControlErrors/Test_SizeControlStates.cpp

```diff
@@ -14,11 +14,11 @@
 
 namespace {
 void test_size_control_update_work(
-    const gsl::not_null<SizeControlInfo*> info,
-    const SizeControlInfo& expected_info,
-    const SizeControlState& size_control_state,
-    const SizeControlStateUpdateArgs& update_args,
-    const CrossingTimeInfo& crossing_time_info) {
+    const gsl::not_null<control_system::size::Info*> info,
```

Comment:
[Optional] Put the whole test in the `control_system::size` namespace so you don't have to keep retyping it? I'm fine either way.